.travis.yml

```diff
@@ -1,9 +1,17 @@
 sudo: required
 
-language: ruby
-
 services:
   - docker
 
-script:
-  - cd hack && PREFIX="openshift/origin-" ./build-images.sh
+matrix:
+  include:
+    - language: ruby
+      script:
+      - cd hack && PREFIX="openshift/origin-" ./build-images.sh
```

Comment:
I don't think we need build images here

Reply:
its a sanity check... but we do that in our CI anyway and could remove.

Reply:
yeah - I think having image build here runs counter to being able to run the unit tests quickly